src/app/components/elements/Voting.jsx

```diff
@@ -141,7 +152,6 @@ class Voting extends React.Component {
             }
             this.setState({ showWeight: !this.state.showWeight });
         };
-        this.shouldComponentUpdate = shouldComponentUpdate(this, 'Voting');
```

Comment:
is this component is `connect`ed with react redux & that is doing an acceptable job of managing `shouldComponentUpdate`?

Reply:
I actually just moved this line up to be with the other React-Lifecycle methods. See Line 73 above...

Reply:
ah, oops